File: lib/src/features/words/models/word.dart
class Word {
  const Word({
    required this.id,
    required this.traductions,
    required this.phonetics,
    required this.userId,
  });

  final String id;
  final Map<String, String> traductions;
  final Map<String, String> phonetics;
  final String userId;
}

File: lib/src/server/models/Word.js
const mongoose = require("mongoose");

const wordSchema = mongoose.Schema({
    trad: {
        type: Object, require: true
    },
    userId: { type: String, require: true },
    definitions: { type: Object, require: false },
    phonetic: { type: Object, require: true }
}, { versionKey: false })

module.exports = mongoose.model('Word', wordSchema);